webapp/src/redux/selectors.ts

```diff
@@ -214,6 +214,19 @@ export const getAreas = createSelector(getStudySynthesis, (synthesis) => {
   return [];
 });
 
+export const getAreasById = createSelector(getStudySynthesis, (synthesis) => {
+  if (synthesis) {
+    return Object.keys(synthesis.areas).reduce(
+      (acc, id) => ({
+        ...acc,
+        [id]: { ...synthesis.areas[id], id },
```

Comment:
It's more performant to mutate in a reduce, it's not necessary to create a new object:

```
acc[id] = ...
return acc
```